jina/serve/streamer.py

```diff
@@ -221,3 +221,34 @@ def get_streamer():
     @staticmethod
     def _set_env_streamer_args(**kwargs):
         os.environ['JINA_STREAMER_ARGS'] = json.dumps(kwargs)
+
+
+class ExecutorStreamer:
+    def __init__(self, connection_pool: GrpcConnectionPool) -> None:
+        self._connection_pool: GrpcConnectionPool = connection_pool
+
+    def __getitem__(self, executor_name: str):
```

Comment:
also we wanted to have something `called` `post` to unify with the interface

Reply:
So the API is like this?:
```python
da = await self.executor['executor1'].post(exec_endpoint='/', docs=DocumentArray([Document(text=text)]), parameters={'k': 'v'})
```

Reply:
implemented this API